src/pretalx/submission/forms/submission.py

```diff
@@ -47,6 +47,10 @@ def __init__(self, event, **kwargs):
                 (a, locale_names[a]) for a in self.event.locales
             ]
 
+        # currently we only display this.
+        # changes only allowed from orga backend.
+        self.fields['slot_count'].disabled = True
```

Comment:
I'm not sure we should even display this field to speakers if they can't set it – that's just frustrating. Displaying it in the organiser area should be enough, I'd say.

Reply:
i think a better solution is to display it and allow modifications until the submission is accepted/confirmed.
that is the same behavior as with the 'withdraw' option. i have extended this text and also fixed the disabled handling and added a user text to the help_text of the field..